flink-runtime/src/main/java/org/apache/flink/runtime/util/OperatorSubtaskDescriptionText.java

```diff
@@ -29,6 +32,9 @@
 	/** Cached description result. */
 	private final String description;
 
+	/** Pattern used to extract operatorId. */
+	public static final Pattern EXTRACT_OPERATOR_PATTERN = Pattern.compile("^.*_([0-9a-f]+)_\\([1-9]+/[1-9]+\\)$");
```

Comment:
Why zero is not taken into account when matching the subtask index and task number?

Reply:
Thanks for your review, the implementation has been changed.